File: src/main/java/com/pei/dehaze/model/query/DictTypePageQuery.java
package com.pei.dehaze.model.query;


import com.pei.dehaze.common.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Schema(description ="字典类型分页查询对象")
@Data
@EqualsAndHashCode(callSuper = true)
public class DictTypePageQuery extends BasePageQuery {

    @Schema(description="关键字(类型名称/类型编码)")
    private String keywords;

}

File: dehaze-go/model/query/dict_type_page_query.go
package query

// DictTypePageQuery 字典类型分页查询对象
type DictTypePageQuery struct {
	// 关键字(类型名称/类型编码)
	Keywords string `json:"keywords"`
	// 页码
	PageNum int `json:"pageNum"`
	// 每页条数
	PageSize int `json:"pageSize"`
}
